From 48f94abff2fcc4943626a62c6180c60862288b08 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fridrich=20=C5=A0trba?= <fridrich.strba@bluewin.ch>
Date: Tue, 3 Jun 2025 11:57:29 +0200
Subject: [PATCH] Install all files generated by doxygen

Install the whole directory and do not pick only some file-types

Change-Id: I743253696f3e57dfe70c6509b7d620dc2bf39e76
---
 docs/doxygen/Makefile.am | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/docs/doxygen/Makefile.am b/docs/doxygen/Makefile.am
index 4351042..40a02e8 100644
--- a/docs/doxygen/Makefile.am
+++ b/docs/doxygen/Makefile.am
@@ -6,9 +6,7 @@ if WITH_LIBZMF_DOCS
 
 install-data-am:
 	mkdir -p $(DESTDIR)$(docdir)/html
-	$(INSTALL_DATA) html/*.html $(DESTDIR)$(docdir)/html/
-	$(INSTALL_DATA) html/*.png $(DESTDIR)$(docdir)/html/
-	$(INSTALL_DATA) html/*.css $(DESTDIR)$(docdir)/html/
+	$(INSTALL_DATA) html/* $(DESTDIR)$(docdir)/html/
 
 uninstall-am:
 	-rm -rf $(DESTDIR)$(docdir)/html
-- 
2.49.0

